### PR TITLE
download certificate

### PR DESCRIPTION
**Pull request by PiranavanShanmugavadivelu**
https://github.com/EmpowerPlastic/empower-platform/issues/5965

### CODE REVIEW

**Review by ba1ciu — COMMENTED** [review]
I'll leave it to @jschill and @portlandnorway

File: frontend/marketplace/src/components/CertificateCard.vue [hunk 1]
```diff
@@ -16,17 +39,427 @@ const certificate = computed(() => {
...
   return props.cardData[1];
 });
-// Define the events that this component can emit
-const emit = defineEmits(["viewCertificate"]);
 
-// Function to emit the 'viewCertificate' event
-const viewCertificate = () => {
-  if (certificate.value.id) {
-    emit("viewCertificate", certificate.value.id);
+const router = useRoute();
```
Comment (jschill) [inline frontend/marketplace/src/components/CertificateCard.vue:43]: Not used

File: frontend/marketplace/src/components/CertificateCard.vue [hunk 2]
```diff
@@ -16,17 +39,427 @@ const certificate = computed(() => {
...
+const { notifyer } = useNotifyer();
+
+const certificateData = ref();
+const creditData = ref();
+const loading = ref(true);
```
Comment (jschill) [inline frontend/marketplace/src/components/CertificateCard.vue:48]: Not used

File: frontend/marketplace/src/pdfGenerator/pdfGenerator.ts [hunk 3]
```diff
@@ -52,7 +52,8 @@ export const generatePDF = (
   applicantDataDescription: string,
   qrCodeUrl: string | undefined,
 ) => {
-  const doc = new jsPDF("landscape") as IjsPDF;
+  //To compress the PDF size
+  const doc = new jsPDF("landscape", undefined, undefined, true) as IjsPDF;
```
Comment (jschill) [inline frontend/marketplace/src/pdfGenerator/pdfGenerator.ts:56]: Great finding! My pdf decreased from 18.1MB to 1.5MB

**Review by jschill — CHANGES_REQUESTED** [review]

**Review by jschill — APPROVED** [review]
👍

**Pull request merged**